### PR TITLE
MapView: Choosing not head local branch to delete.

### PR DESCRIPTION
**Pull request by lolgear**
- [x] #166

### CODE REVIEW

**Comment by lucasderraugh** [comment]
Do you think this might result in accidental deletion of the wrong branch? Do you think we need to make it more explicit that it's not the branch they're currently on?
I think it may be fine as is, just wanted your thoughts on it 😄.

**Comment by lucasderraugh** [comment]
My biggest concern is that you'd inadvertently delete a branch that you're not actually on. I think without a warning to explicitly say you're deleting a specific branch, I think this is too dangerous.

**Comment by lolgear** [comment]
@lucasderraugh 
As I remember, It doesn't delete branch without dialog. But now there is no way to delete a branch `Qool_feature` that is merged into `master`. Alphabetical ordering will always choose `master`.

File: GitUpKit/Views/GIMapViewController.m [hunk 1]
```diff
@@ -935,9 +935,25 @@ - (IBAction)revertSelectedCommit:(id)sender {
   [self revertCommit:commit againstLocalBranch:self.repository.history.HEADBranch];
 }
 
+- (GCHistoryLocalBranch *)branchToDeleteForSelectedCommit:(GCHistoryCommit *)commit {
+  // return commit.localBranches.firstObject;
+  NSArray <GCHistoryLocalBranch *>* localBranches = commit.localBranches;
+//  NSArray <GCHistoryRemoteBranch *> *remoteBranches = commit.remoteBranches;
```
[the remark this answers is not in the document]
Comment (lolgear) [reply GitUpKit/Views/GIMapViewController.m]: @lucasderraugh Done.

**Comment by lucasderraugh** [comment]
Thanks @lolgear. Looks great! Sorry I've taken so long to work on this I'm just pretty busy with my job that pays 😅

**Pull request merged**